### PR TITLE
add sgx-ansible script (Ubuntu 20.04)

### PR DESCRIPTION
**Pull request by haerdib**
Adds the sgx-ansible script for Ubuntu 20.04 with an Example User ssh key and and examplehost server. It's up the up to date script with our current, unpublished version.

I have skimmed all files but didn't find any personal information left. What I did find however, is that some scripts (which we have not written by ourselves, but rather imported from geerlinguy) are outdated and will most likely not work for Ubuntu 20.04. These scripts are unrelated to sgx itself.

Should we maintain these scripts, leave as is, or delete them? (example: keyboard, munin-node, netplan...)

The README will be added with a separate PR.

Related to issue #1

### CODE REVIEW

File: sgx-ansible/roles/role-geerlingguy.certbot/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+env:
+  global:
+    - ROLE_NAME: certbot
+  matrix:
+    - MOLECULE_DISTRO: centos8
+    - MOLECULE_DISTRO: centos7
+    - MOLECULE_DISTRO: centos6
+      MOLECULE_PLAYBOOK: playbook-source-install.yml
+    - MOLECULE_DISTRO: ubuntu1804
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.certbot/.travis.yml:13]: outdated

File: sgx-ansible/roles/role-certificate/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  company: https://botka.link
+  github_branch: master
+  license: BSD
+  min_ansible_version: 2.7
+  platforms:
+    - name: FreeBSD
+      versions:
+        - 11.2
+    - name: Ubuntu
+      versions:
```
Comment (haerdib) [inline sgx-ansible/roles/role-certificate/meta/main.yml:14]: outdated ubuntu version

File: sgx-ansible/roles/role-bertvv.samba/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# .travis.yml Execution script for role tests on Travis-CI
+---
+sudo: required
+
+env:
+  matrix:
+    - DISTRIBUTION: centos
+      VERSION: 7
+    - DISTRIBUTION: ubuntu
+      VERSION: 18.04
```
Comment (haerdib) [inline sgx-ansible/roles/role-bertvv.samba/.travis.yml:10]: outdated ubuntu version

File: sgx-ansible/roles/role-geerlingguy.docker/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+language: python
+services: docker
+
+env:
+  global:
+    - ROLE_NAME: docker
+  matrix:
+    - MOLECULE_DISTRO: centos8
+    - MOLECULE_DISTRO: centos7
+    - MOLECULE_DISTRO: ubuntu1804
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.docker/.travis.yml:11]: outdated

File: sgx-ansible/roles/role-geerlingguy.docker/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        - 8
+    - name: Fedora
+      versions:
+        - all
+    - name: Debian
+      versions:
+        - stretch
+        - buster
+    - name: Ubuntu
+      versions:
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.docker/meta/main.yml:23]: outdated

File: sgx-ansible/roles/role-geerlingguy.gitlab/.travis.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+env:
+  global:
+    - ROLE_NAME: gitlab
+  matrix:
+    - MOLECULE_DISTRO: centos7
+    - MOLECULE_DISTRO: ubuntu1804
+    - MOLECULE_DISTRO: debian9
+    - MOLECULE_DISTRO: centos7
+      MOLECULE_PLAYBOOK: playbook-version.yml
+    - MOLECULE_DISTRO: ubuntu1804
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.gitlab/.travis.yml:14]: outdated

File: sgx-ansible/roles/role-geerlingguy.munin-node/.travis.yml [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+services: docker
+
+env:
+  # Test general usage.
+  - distro: centos7
+    playbook: test.yml
+  - distro: ubuntu1604
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.munin-node/.travis.yml:8]: outdated

File: sgx-ansible/roles/role-geerlingguy.nginx/meta/main.yml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+      versions:
+        - 6
+        - 7
+    - name: Debian
+      versions:
+        - all
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
```
Comment (haerdib) [inline sgx-ansible/roles/role-geerlingguy.nginx/meta/main.yml:21]: outdated

File: sgx-ansible/roles/role-intel-sgx/meta/main.yml [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: license (GPL-2.0-or-later, MIT, etc)
```
Comment (haerdib) [inline sgx-ansible/roles/role-intel-sgx/meta/main.yml:17]: Is this one still up to date @brenzi ?

File: sgx-ansible/roles/role-mrlesmithjr.netplan/meta/main.yml [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+  license: MIT
+
+  min_ansible_version: 1.2
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - artful
+        - bionic
```
Comment (haerdib) [inline sgx-ansible/roles/role-mrlesmithjr.netplan/meta/main.yml:14]: outdated

File: sgx-ansible/roles/role-intel-sgx/meta/main.yml (hunk 9 above)
Comment (brenzi) [reply sgx-ansible/roles/role-intel-sgx/meta/main.yml:17]: don.t know

**Comment by brenzi** [comment]
If the basic installation works as-is, I'd suggest to merge this one and just file an issue to update the outdated dependencies

**Pull request merged**